src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticSourceEventSource.cs

```diff
@@ -81,6 +82,50 @@ namespace System.Diagnostics
     /// into the output.   This is not likely to be a good idea as it will be very verbose, but is useful to quickly
     /// discover what is available.
     /// 
+    /// 
+    /// * How data is loged in the EventSource 
+    /// 
+    /// By default all data from Diagnostic sources is logged to the the DiagnosticEventSouce event called 'Event' 
+    /// which has three fields  
+    /// 
+    ///     string SourceName, 
+    ///     string EventName, 
+    ///     IEnumerable<KeyValuePair<string, string>> Argument
+    /// 
+    /// However to spport start-stop activity tacking, there are six other events that can be used 
+    /// 
+    ///     Activity1Start         
+    ///     Activity1Stop
+    ///     Activity2Start
+    ///     Activity2Stop
```

Comment:
Why are there two sets?  Is it so that multiple activities can be tracked at the same time, and if so, how did we settle on 2 being the right number of concurrent activities to enable?